packages/server-wallet/src/engine/engine.ts

```diff
@@ -64,95 +64,93 @@ import {
   SingleChannelOutput,
   MultipleChannelOutput,
   Output,
-  WalletInterface,
+  EngineInterface,
   UpdateChannelFundingParams,
-  WalletEvent,
+  EngineEvent,
 } from './types';
-import {WalletResponse} from './wallet-response';
+import {EngineResponse} from './engine-response';
 
 // TODO: The client-api does not currently allow for outgoing messages to be
 // declared as the result of a wallet API call.
 // Nor does it allow for multiple channel results
 
 type EventEmitterType = {
-  [key in WalletEvent['type']]: WalletEvent['value'];
+  [key in EngineEvent['type']]: EngineEvent['value'];
 };
 
 export class ConfigValidationError extends Error {
   constructor(public errors: ValidationErrorItem[]) {
-    super('Server wallet configuration validation failed');
+    super('Engine configuration validation failed');
   }
 }
 
 /**
- * A single-threaded Nitro wallet
```

Comment:
I don't really know what a Nitro engine is, but that's probably ok 🤷

Reply:
Haven't you seen fast and furious?